sonic_platform_base/sonic_sfp/ext_media_utils.py

```diff
@@ -0,0 +1,165 @@
+########################################################################
+# DellEMC
+#
+# Module contains common classes and utilities for other components
+# of the extended media functionality
+#
+########################################################################
+
+DEFAULT_NO_DATA_VALUE = 'N/A'
+
+# Connector codes are common to all form-factors
+connector_code_to_name_map = {
```

Comment:
we already have this defined in both parser SFF8472 and SFF8436, can I suggest to extend and reuse?

SFF8436
```

    connector = {
            '00': 'Unknown or unspecified',
            '01': 'SC',
            '02': 'FC Style 1 copper connector',
            '03': 'FC Style 2 copper connector',
            '04': 'BNC/TNC',
            '05': 'FC coax headers',
            '06': 'Fiberjack',
            '07': 'LC',
            '08': 'MT-RJ',
            '09': 'MU',
            '0a': 'SG',
            '0b': 'Optical Pigtail',
            '0c': 'MPOx12',
            '0d': 'MPOx16',
            '20': 'HSSDC II',
            '21': 'Copper pigtail',
            '22': 'RJ45',
            '23': 'No separable connector'
            }
```

SFF8472

```
    connector = {'00': 'Unknown',
             '01': 'SC',
             '02': 'Fibre Channel Style 1 copper connector',
             '03': 'Fibre Channel Style 2 copper connector',
             '04': 'BNC/TNC',
             '05': 'Fibre Channel coaxial headers',
             '06': 'FibreJack',
             '07': 'LC',
             '08': 'MT-RJ',
             '09': 'MU',
             '0a': 'SG',
             '0b': 'Optical pigtail',
             '0c': 'MPO Parallel Optic',
             '20': 'HSSDCII',
             '21': 'CopperPigtail',
             '22': 'RJ45'}
```